.github/workflows/sync_branches.yml

```diff
@@ -39,12 +55,12 @@ jobs:
         run: |
           if git fetch origin "${SYNC_BRANCH}"; then
             echo "Branch ${SYNC_BRANCH} already exists, checking if the branch was modified..."
-            COMMITTER="$(git show --pretty=format:"%cn <%ce>" --no-patch "origin/${SYNC_BRANCH}")"
-            if [ "${COMMITTER}" = "${SYNC_COMMITTER}" ]; then
+            echo "branch_exists=true" >> $GITHUB_OUTPUT
+            COMMITTER_EMAIL="$(git show --pretty=format:"%ce" --no-patch "origin/${SYNC_BRANCH}")"
+            if [ "${COMMITTER_EMAIL}" = "${SYNC_COMMITTER_EMAIL}" ]; then
               echo "Branch ${SYNC_BRANCH} was not modified."
             else
               echo "Branch ${SYNC_BRANCH} was modified, either delete it or update it yourself."
```

Comment:
Are these messages outdated? What shall happen if there has been a conflict resolved?

Reply:
Yep, good point. The expected behaviour is the automation to try and keep the branch in sync with further change that may have happen after the manual resolution, in a non destructive way (basically pushing further commits, if no new conflict have appeared)